semantics/language/common/expr/eval.k

```diff
@@ -41,19 +43,19 @@ module C-COMMON-EXPR-EVAL
      }*/
 
      // Lvalue conversion.
-     rule reval(nclv(Loc:SymLoc, T:Type)
+     rule reval(nclv(Loc::SymLoc, T::Type)
```

Comment:
This is not just a matter of performance. If you work with klabels, you can easily violate the signature. Also for other backends may work better with the sort information. You are skewing the C semantics towards the OCaml backend. How would you feel if I started to change the semantics to make the verification faster or Brandon to make the Coq backend work better?

If the performance is so important, I suggest trying to cache the predicates, use OCaml datatypes, or otherwise accept this as a limitation of the OCaml backend.